File: gracie.lee/src/main/java/homework/chapter_6/homework1/Member.java
package homework.chapter_6.homework1;

import jakarta.persistence.*;

@Entity(name = "Hw1Member")
@Table(name = "member_hw1")
public class Member {

    @Id
    @GeneratedValue
    @Column(name = "member_id")
    private Long id;

    private String name;

    // 일대다 단방향: Member는 Team을 참조하지 않음!
    // (Team → Member 방향만 존재)

    protected Member() {
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: gracie.lee/src/main/kotlin/homework/chapter_6/homework1/Member.kt
package homework.chapter_6.homework1

import jakarta.persistence.*

@Entity(name = "Hw1MemberKt")
@Table(name = "member_hw1_kt")
class Member(
    @Id
    @GeneratedValue
    @Column(name = "member_id")
    val id: Long? = null,

    var name: String? = null
)
// 일대다 단방향: Member는 Team을 참조하지 않음!
// (Team -> Member 방향만 존재)
